tools/tidyall

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env perl
+#
+# Tidyall command with perltidy version constraint.
+use Perl::Tidy;
+use strict;
+use warnings "all";
+
+use constant REQUIRED_VERSION => '20230912';
+my $detected_version = $Perl::Tidy::VERSION;
+my @tidyall_argv = @ARGV;
+
+sub is_force_flag { $_ eq '--force' }
+
+unless ($detected_version == REQUIRED_VERSION) {
+    print  STDERR "Incorrect version of perltidy.\n";
+    printf STDERR "- Detected: %s\n+ Required: %s\n\n", $detected_version, REQUIRED_VERSION;
+
+    my $force_run = grep { is_force_flag } @ARGV;
+
+    unless ($force_run) {
+        printf STDERR "Please install the appropriate version of perltidy.\n";
+        printf STDERR "If you want to proceed anyways, re run with --force flag.\n";
+        exit 1;
+    }
+
+    # tidyall does not know about the --force flag.
+    @tidyall_argv = grep { !is_force_flag } @tidyall_argv;
+
+    print STDERR "Proceeding to run with incorrect version of perltidy. ";
+    print STDERR "Results might not be consistent.\n";
+    print STDERR "==================\n";
+}
+
+exit exec('tidyall', @tidyall_argv);
```

Comment:
```suggestion
exec 'tidyall', @tidyall_argv;
```
The exec will replace the current process, so the `exit` itself won't be executed anyway.